### PR TITLE
Add support for Xiaomi Mi 5

### PR DESCRIPTION
**Pull request by sib0ndt**
Add support for Xiaomi Mi 5 (gemini)

### CODE REVIEW

File: lk2nd/device/dts/msm8996/msm8996-xiaomi-mi5.dts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8996 0x30001>;
+	qcom,board-id = <31 0>, <32 0>;
+	qcom,pmic-id = <0x20009 0x2000a 0x00 0x00>;
```
Comment (barni2000) [inline lk2nd/device/dts/msm8996/msm8996-xiaomi-mi5.dts:9]: Does `pmic-id` necessary?

Comment (sib0ndt) [reply lk2nd/device/dts/msm8996/msm8996-xiaomi-mi5.dts:9]: i think so,

**Comment by TravMurav** [comment]
Thanks! I've manually squashed commits and reworded the subject. I'd appreciate changes like this to be a single commit with correct subject next time

File: lk2nd/device/dts/msm8996/msm8996-xiaomi-mi5.dts (hunk 1 above)
Comment (barni2000) [reply lk2nd/device/dts/msm8996/msm8996-xiaomi-mi5.dts:9]: I think it could boot with out it but nvm it was already merged

**Pull request merged**